scripts/components/Interactions/HotspotNavButton.js

```diff
@@ -39,6 +39,21 @@ export default class HotspotNavButton extends React.Component {
     });
   }
 
+  /**
+   * React life-cycle handler: Component did update.
```

Comment:
@henriettemoe Thanks for adopting these comments. I'd agree that they are not required, but the goal is to make it as simple for newcomers as possible.